Gems/AtomLyIntegration/AtomImGuiTools/Code/Source/AtomImGuiToolsSystemComponent.cpp

```diff
@@ -111,6 +111,11 @@ namespace AtomImGuiTools
     {
         if (ImGui::BeginMenu("Atom Tools"))
         {
+            if (ImGui::MenuItem("Dump loaded Asset info", ""))
+            {
+                AZ::Data::AssetManager::Instance().DumpLoadedAssetsInfo();
```

Comment:
Could we dump this using a console command similar to sys_DumpAllocators?

Reply:
We have an issue that the cvar callback might be called several times. 
Also, our console in game launcher is implemented with ImGui. So if you have access to console, you would have the access to ImGui menu item.